website/website/assets/js/map-en.js

```diff
@@ -222,6 +222,7 @@ function buildMap(...charts){
 			  "dataset": String(selectedDataset),
 			  "featureId": String(areas),
 			  "includeSvg": true
+			  "returnHTML": false
```

Comment:
```suggestion
			  "includeHtml": false
```
Just to be consistent with `includeSvg`